task1/index.js

```diff
@@ -0,0 +1,21 @@
+let arrCity = ["Київ", "Вінниця", "Миколаїв", "Чернігів"];

```

Comment:
Hi there)) It's a best practice to use const instead of let everywhere where it's possible because it gives awareness that variable contains the same type during the execution. So please change all declarations